src/constants/infra.ts

```diff
@@ -0,0 +1,2 @@
+// NOTE: 1 for aws load balancer and another for cloudflare reverse proxy.
+export const NUM_PROXIES = 2
```

Comment:
minor nit: shall we use default export?

Reply:
think i'd rather not here - we should default export for `NamedFiles`, which contain a primary export and potentially other secondary exports (eg: our routers/services in be or our components/pages in fe) but for constants, they should all be equal (+ we might add more here in future) so would rather not